src/graphnet/models/gnn/dynedge.py

```diff
@@ -65,6 +66,10 @@ def __init__(
                 after global pooling. The alternative is to  added (distribute)
                 them to the individual nodes before any convolutional
                 operations.
+            icemix_encoder: Whether to use the IceCubeMix encoder. If `True`,
```

Comment:
I think we should split this into three separate args:

```python
activation : Callable = torch.nn.LeakyReLU()
add_layer_norm: bool = False
skip_readout: bool = False
```
The kaggle-version of this model would then be a specific set of args, i.e `activation = torch.nn.GELU()`, `add_layer_norm = True` and `skip_readout = True`.